Sky Atlas/Sky Atlas.html

```diff
@@ -19115,6 +19107,13 @@ <h1>Sections & Primary Docs</h1>
             <li>Determine Target Reward Value: Calculate the total US dollar value required to meet the SSR for the period: (Total Eligible USDS Balance * SSR).</li>
             <li>Determine SKY Token Quantity: Divide the Target Reward Value (from step 1) by the Current Market Price of SKY (determined at the time of the Executive Vote).</li>
           </ol>
+          <p>The Governance Facilitators, on the recommendation of the Stability Scope Advisors, may maintain the quantity of SKY token rewards to be distributed for a normalization period at its current level if they conclude that:</p>
+          <ul>
+            <li>the difference between the effective yield provided by SKY Tokens Rewards and the SSR is not material;</li>
+            <li>the difference between the effective yield provided by SKY Token Rewards and the SSR is unlikely to persist due to other factors; or</li>
+            <li>adjusting the quantity of SKY Token Rewards is otherwise inadvisable.</li>
+          </ul>
+          <p>The recommendation of the Stability Scope Advisors must be posted to the Sky Forum and approved by the Stability Facilitators.</p>
```

Comment:
nit: you don't need p tags around lists, they are implied.

```suggestion
          The Governance Facilitators, on the recommendation of the Stability Scope Advisors, may maintain the quantity of SKY token rewards to be distributed for a normalization period at its current level if they conclude that:
          <ul>
            <li>the difference between the effective yield provided by SKY Tokens Rewards and the SSR is not material;</li>
            <li>the difference between the effective yield provided by SKY Token Rewards and the SSR is unlikely to persist due to other factors; or</li>
            <li>adjusting the quantity of SKY Token Rewards is otherwise inadvisable.</li>
          </ul>
         The recommendation of the Stability Scope Advisors must be posted to the Sky Forum and approved by the Stability Facilitators.
```